<launch>
  <node name="slamkitNode"          pkg="slamkit_ros"  type="slamkitNode" output="screen">
    <param name="channel_type"         type="string" value="usb"/>  
    <param name="frame_id"             type="string" value="imu"/> 
    <param name="usb_venderId_slamkit"     type="int"    value="64719"/>
    <param name="usb_productId_slamkit"    type="int"    value="61696"/>
    <param name="usb_interfaceId_slamkit"  type="int"    value="3"/>
    <param name="usb_txEndpoint_slamkit"   type="int"    value="5"/>
    <param name="usb_rxEndpoint_slamkit"   type="int"    value="5"/>
 </node>
 
  <node name="complementary_filter_node"          pkg="imu_complementary_filter"  type="complementary_filter_node" output="screen">
    <param name="publish_debug_topics "   type="bool"    value="true"/>
    <param name="gain_acc "   type="double"    value="0.01"/>
  </node>

  <node name="slamkitNodeClient"    pkg="slamkit_ros"  type="slamkitNodeClient" output="screen">
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find slamkit_ros)/rviz/imu_display.rviz" />
</launch>
